lib/src/utils/serialization_utils.dart

```diff
@@ -128,6 +137,20 @@ class SerializationUtils {
     });
   }
 
+  static Uint8List serializeCoin(String coin) {
+    return useBinaryWriter((ByteDataWriter writer) {
+      writeSerializedUint64(writer, BigInt.parse(coin));
+      return writer.toBytes();
+    });
+  }
+
+  static Uint8List serializeTxTtl(String ttl) {
+    return useBinaryWriter((ByteDataWriter writer) {
+      writeSerializedUint64(writer, BigInt.parse(ttl));
+      return writer.toBytes();
+    });
+  }
```

Comment:
ttl ar trebui sa fie de asemenea BigInt. Si la fel ar trebui ceva functie care scrie bigint fara sa faca limita la Uint64

Reply:
Esti sigur ca acest ttl nu e serializat direct ca String in ts? 

De ex: Daca serializezi numarul 20 ca Uint sau "20" ca string o sa dea un rezultat complet diferit.

Reply:
Acelasi lucru aici. `ttl` ar trebui sa fie din start BigInt